app/views/dashboard/index.html.erb

```diff
@@ -0,0 +1,396 @@
+          }
+        });
+      }
+    });
+
+    let chartBar = new Chartist.Bar('#chart3', {
+      labels: ['Q1', 'Q2', 'Q3', 'Q4'],
+      series: [
+        [800000, 1200000, 1400000, 1300000],
+        [200000, 400000, 500000, 300000],
+        [100000, 200000, 400000, 600000]
+      ]
+    }, {
+      stackBars: true,
+      axisY: {
+        labelInterpolationFnc: function (value) {
+          return (value / 1000) + 'k';
+        }
+      }
+    })
+
+    chartBar.on('draw', function (data) {
+      if (data.type === 'bar') {
+        data.element.attr({
+          style: 'stroke-width: 30px'
+        }),
+          data.element.animate({
+            y2: {
+              dur: '0.5s',
+              from: data.y1,
+              to: data.y2
+            }
+          });
+      }
+    });
+
+    let chartPie = new Chartist.Pie('#chart4', {
+      series: [10, 20, 50, 20, 5, 50, 15],
+      labels: [1, 2, 3, 4, 5, 6, 7]
+    }, {
+      donut: true,
+      showLabel: true
+    });
+
+    chartPie.on('draw', function (data) {
+      if (data.type === 'slice') {
+        let pathLength = data.element._node.getTotalLength();
+        data.element.attr({
+          'stroke-dasharray': pathLength + 'px ' + pathLength + 'px'
+        });
+
+        let animationDefinition = {
+          'stroke-dashoffset': {
+            id: 'anim' + data.index,
+            dur: 200,
+            from: -pathLength + 'px',
+            to: '0px',
+            easing: Chartist.Svg.Easing.easeOutQuint,
+            fill: 'freeze'
+          }
+        };
+
+        if (data.index !== 0) {
+          animationDefinition['stroke-dashoffset'].begin = 'anim' + (data.index - 1) + '.end';
+        }
+
+        data.element.attr({
+          'stroke-dashoffset': -pathLength + 'px'
+        });
+
+        data.element.animate(animationDefinition, false);
+      }
+    });
+
+    /*Toggle dropdown list*/
+    /*https://gist.github.com/slavapas/593e8e50cf4cc16ac972afcbad4f70c8*/
+
+    let userMenuDiv = document.getElementById("userMenu");
+    let userMenu = document.getElementById("userButton");
+
+    document.onclick = check;
+
+    function check(e) {
+      let target = (e && e.target) || (event && event.srcElement);
+
+      //User Menu
+      if (!checkParent(target, userMenuDiv)) {
+        // click NOT on the menu
+        if (checkParent(target, userMenu)) {
+          // click on the link
+          if (userMenuDiv.classList.contains("invisible")) {
+            userMenuDiv.classList.remove("invisible");
+          } else {
+            userMenuDiv.classList.add("invisible");
+          }
+        } else {
+          // click both outside link and outside menu, hide menu
+          userMenuDiv.classList.add("invisible");
+        }
+      }
+
+    }
+
+    function checkParent(t, elm) {
+      while (t.parentNode) {
+        if (t == elm) {
+          return true;
+        }
+        t = t.parentNode;
+      }
+      return false;
+    }
```

Comment:
This JS code, put it in an external file.

Reply:
done